charts/dolos/templates/sts.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "dolos.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
This should be in the pod template annotations. A change here won't replace the pod.